client-reactjs/src/components/application/Jobs/JobDetails.js

```diff
@@ -471,180 +313,30 @@ class JobDetails extends Component {
 
   onClusterSelection = (value) => {
     this.props.dispatch(assetsActions.clusterSelected(value));
-    this.setState({
-      selectedCluster: value,
-    });
-  };
-
-  searchJobs(searchString) {
```

Comment:
searchJobs is removed from here but it does not seem to be added to another component.